src/main/java/com/denizenscript/clientizen/util/Utilities.java

```diff
@@ -0,0 +1,12 @@
+package com.denizenscript.clientizen.util;
+
+public class Utilities {
+
+    public static float normalizeYaw(float yaw) {
```

Comment:
Why not a static method on LocationTag?

Reply:
Fair enough, moved it to `LocationTag` for now